tests/context/context_constructors.cpp

```diff
@@ -66,56 +66,77 @@ class TEST_NAME : public util::test_base {
 
   /** execute the test
    */
-  void run(util::logger &log) override {
+  void run(util::logger& log) override {
     cts_async_handler asyncHandler;
-
+    sycl::property_list property_list{};
     {
-      /** check default constructor and destructor
+      /** check default constructor, destructor
+       *  and (const property_list&) constructor
```

Comment:
Useless since we have color screens and no more punchcards :-)
```suggestion
          and (const property_list&) constructor
```